common/djangoapps/student/management/commands/change_enrollment.py

```diff
@@ -70,19 +70,21 @@ def handle(self, *args, **options):
             mode=options['from_mode']
         )
         if options['user']:
-            if '@' in options['user']:
-                user = User.objects.get(email=options['user'])
-            else:
-                user = User.objects.get(username=options['user'])
-            filter_args['user'] = user
-        enrollments = CourseEnrollment.objects.filter(**filter_args)
-        if options['noop']:
-            print "Would have changed {num_enrollments} students from {from_mode} to {to_mode}".format(
-                num_enrollments=enrollments.count(),
-                from_mode=options['from_mode'],
-                to_mode=options['to_mode']
-            )
-        else:
-            for enrollment in enrollments:
-                enrollment.update_enrollment(mode=options['to_mode'])
-                enrollment.save()
+            user_str = options['user']
+            for username in user_str.split(","):
+                if '@' in username:
+                    user = User.objects.get(email=username)
+                else:
+                    user = User.objects.get(username=username)
+                filter_args['user'] = user
+                enrollments = CourseEnrollment.objects.filter(**filter_args)
```

Comment:
Looks like you indented everything below here an extra step. Was that intentional?